substratevm/src/com.oracle.svm.test/src/com/oracle/svm/test/jfr/TestGarbageCollectionEvents.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, 2022, Red Hat Inc. All rights reserved.
```

Comment:
I will change the copyright years of this file to 2025 if this is fine with you.